alleycats-core/src/main/scala/alleycats/OrElse.scala

```diff
@@ -0,0 +1,8 @@
+package alleycats
+
+import simulacrum.op
+import simulacrum.typeclass
+
+@typeclass trait OrElse[F[_]] {
+  @op("orElse") def orElse[A](fa: F[A], alternative: => F[A]): F[A]
```

Comment:
I think `@op def ...` will work the same :)

Reply:
Truly!

Reply:
hmm, interesting. I was pretty sure it had at least an overload that doesn't take a name. Sorry :)